app/views/promo_mailer/new_channel_registered_2018q1.html.slim

```diff
@@ -0,0 +1,18 @@
+- content_for(:title) do
+  h3= t("promo_mailer.new_channel_registered_2018q1.subject")
+
+p.salutation= t("publisher_mailer.shared.salutation", name: @publisher.name)
+
+p = t("promo_mailer.promo_activated_2018q1_verified.body_one")
+
+.promo--item-label = "#{@channel.publication_title}"
+.promo--referral-link-container
+  span.promo--referral-link = referral_link(@channel.promo_registration.referral_code)
+  = link_to("", tweet_url(@channel.promo_registration.referral_code), style: "background-image: url(#{attachments["tweet.png"]&.url})", class: 'promo--share-button')
```

Comment:
- does it work in Gmail?

- `&.url` means these images could fail silently.
if we're to use that, it's important to create a test to confirm the image shows up.

Reply:
This does not consistently work in gmail.  Another thing I noticed is that gmail displays these images only as attachments. ex:

![image](https://user-images.githubusercontent.com/12549658/35985046-b53c2b38-0cc3-11e8-87aa-fbceb0acd468.png)

I do not have a plan to resolve this yet, any input would be much appreciated as this is hard to debug since there is no problem locally.

Reply:
a majority (source: my previous job) of people use gmail so it's necessary if we want to ship the button

maybe try css `background` shorthand? some hints here (could be hearsay)
https://litmus.com/community/discussions/5645-background-images-breaking-in-gmail#comment-8973